app/client/components/navBar/navBar.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from "react";
+import { Mobile, Default } from "../helpers/responsive";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+import { ReduxState } from "../../reducers/rootReducer";
+import { replaceSpaces } from "../../utils";
+import MobileTopBar from "./MobileTopBar";
+import OrgSwitcher from "./OrgSwitcher";
+
+interface StateProps {
+  loggedIn: boolean;
+  email: string | null;
+  orgName: string;
+}
+
+interface DispatchProps {}
+
+const initialState = Object.freeze({
+  iconURL: "/static/media/sempo_icon.svg",
+  mobileMenuOpen: false,
+  isOrgSwitcherActive: false
+});
+
+type Props = DispatchProps & StateProps;
+type State = typeof initialState;
+
+declare global {
```

Comment:
isn't needed here because typescript will merge all related interfaces but editor gets angry when window isn't defined. i think there's probably some support to handle this better